containers/podman-desktop
# fix: refine colors used in form pages

### What does this PR do?

This PR refines the changes made by https://github.com/containers/podman-desktop/pull/7768 . I just recently discovered the mockup made by Emma https://github.com/containers/podman-desktop/wiki/Design-System-Theming#form-page so i converted the classes to what it is defined there. The output is almost the same as the main branch. 

This allowed me to remove the custom formpage classes which are not needed anymore.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

it is part of #7214 

### How to test this PR?

The pages to verify:

1. ExportContainer: go to containers page -> click on the "three vertical dots" icon of a container -> export container
2. ImportImage: go to the images page -> click on import (top right)
3. LoadImages: go to the images page -> click on load(top right)
4. PullImage: go to the images page -> click on pull(top right)
5. RunImage: go to the images page -> click on the start action of an image
6. SaveImages: go to images page -> click on the "three vertical dots" icon of an image-> save image
7. DeployToKube: go to the pods page -> click on the "three vertical dots" icon of an image-> deploy to kube
8. PodCreateFromContainers: go to containers page -> select 2 or more containers -> click on create pod on the top
9. CreateVolume: go to volumes page -> click on create (top right)
